admin/make_homebrew_recipe.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Create a Homebrew recipe for Flocker, using the VERSION environment variable.
+
+Invoke using e.g.:
+  $ export VERSION=0.1.2
+  $ ./make_homebrew_recipe.py > flocker-${VERSION}.rb
+
+Inspired by https://github.com/tdsmith/labmisc/blob/master/mkpydeps.
+"""
+
+from os import environ
+from json import load
+from urllib2 import urlopen
+from hashlib import sha1
+from tl.eggdeps.graph import Graph
+
+version = environ.get("VERSION")
+if version is None:
+    raise Exception("Set the VERSION environment variable.")
+
+url = ("http://storage.googleapis.com/archive.clusterhq.com/"
+       "downloads/flocker/Flocker-{version}.tar.gz").format(version=version)
```

Comment:
> And ideally (I think) we'd just supply the Flocker sdist package as an argument to this script. 
The locally built package is the "known good" package. It just happens to be uploaded to google storage in our current process. 

I see the advantage, but this way tests that the package is actually available at the download URL, so I'd rather keep it with better error reporting.